spec/models/spree/order_decorator_spec.rb

```diff
@@ -90,6 +82,22 @@
     end
   end
 
+
+  describe "#adjust_avalara" do
+    context 'committed order' do
+      let(:adjust_order) {
+        order.avalara_capture_finalize
+        order
```

Comment:
`order.tap {|o| o.avalara_capture_finalize }` and you can have it on a single line